DQMOffline/Hcal/src/HcalRecHitsAnalyzer.cc

```diff
@@ -613,27 +608,24 @@ void HcalRecHitsAnalyzer::analyze(edm::Event const &ev, edm::EventSetup const &i
   //===========================================================================
   // IN ALL other CASES : ieta-iphi maps
   //===========================================================================
-
+  /*
```

Comment:
Since you are removing the lines were eEcal , eEcalB , eEcalE are filled. you should then also remove all the lines that you have commented out, betweem L611 and L628

Reply:
thanks @perrotta , done